include/deal.II/grid/reference_cell.h

```diff
@@ -98,6 +98,32 @@ namespace internal
 class ReferenceCell
 {
 public:
+  /**
+   * Enum of different choices for istropic refinement.
+   * There are 3 different ways to refine a tetrahedral, here we save the
+   * different possibilites. It is different to RefinementPossibilites are these
+   * are options in the case that an isotropic refinement is conducted.
+   */
+  enum IsotropicRefinementChoices : std::uint8_t
```

Comment:
Could we convert this to an `enum class`?

Reply:
Converting it to an `enum class` would make the code harder to read as names would expand considerably from e.g. `ReferenceCell::isotropic_refinement` to `ReferenceCell::IsotropicRefinementChoices::isotropic_refinement` and the implicit conversion to `unsigned char` is often used, which then would not work I guess. But I'm not too familiar with `enum classes`, so there is maybe a elegant way to do it, if not I would prefer to keep it just as `enum`.

Reply:
What about moving `IsotropicRefinementChoices` outside of `ReferenceCell`? I think we nowadays we prefer `enum class` over `enum`.